map/src/main/res/layout/activity_map.xml

```diff
@@ -18,6 +18,23 @@
         app:navGraph="@navigation/map_navigation_graph"
         tools:background="#fab" />
 
+
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="50dp"
+        app:layout_constraintBottom_toTopOf="@id/bottomNavigation"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent">
+
+        <include
+            android:id="@+id/audioPlayer"
+            layout="@layout/minimal_exo_playback_control_view"
+            android:layout_width="match_parent"
+            android:layout_height="50dp"
```

Comment:
DRY. 50dp is specified already above and should be a dimen if we need exact height.

Reply:
updated.